project.clj

```diff
@@ -1,4 +1,4 @@
-(def ps-version "6.0.1-SNAPSHOT")
+(def ps-version "6.0.2-SNAPSHOT")
```

Comment:
Do we want to update this to `6.1.0-SNAPSHOT`?

Reply:
Yeah, I was just thinking about getting the merge up done, but makes sense to do it now.

Reply:
updated